src/main/java/frc/robot/subsystems/auto/AutoLogic.java

```diff
@@ -293,4 +295,13 @@ public static Command isCollected() {
     }
     return Commands.none().withName("isCollected");
   }
+  public static Command readyIntakeCommand() {
+    if (r.superStructure != null) {
+      if (ARMSENSOR_ENABLED) {
+        return Commands.sequence(r.superStructure.coralPreIntake())
+            .withName("readyIntake");
+      }
```

Comment:
```suggestion
      return r.superStructure.coralPreIntake().withName("readyIntake");
```
Moving to pre-intake doesn't require the arm sensor, and we don't need to make a sequence containing one command.